src/components/AddPlaidBankAccount.js

```diff
@@ -111,7 +113,28 @@ class AddPlaidBankAccount extends React.Component {
         return lodashGet(this.props.plaidBankAccounts, 'accounts', []);
     }
 
+    /**
+     * @returns {Object}
+     */
+    getErrors() {
```

Comment:
This looks like its repeated in a few places, can we move it to ReimbursementAccountUtils or something similar?

Reply:
Yes! Thanks for this suggestion. I was envisioning this logic would eventually get abstracted into a component somehow. But until we figure that out what that will look like... probably best to have all of these repeated methods in lib.